bundles/org.openhab.binding.opengarage/README.md

```diff
@@ -34,15 +34,15 @@ As a minimum, the IP address is needed:
 opengarage.things:
 
 ```
-opengarage:opengarage:garage [ host="192.168.0.5" ]
+opengarage:opengarage:OpenGarage [ host="192.168.0.5" ]
```

Comment:
What is wrong with `garage`?

Reply:
Nothing really, but since I started with configuring this through UI, this was the default name. I just wanted to keep it in sync for people who prefers UI.